play_motion:
  motions:
    pregrasp:
      joints: [torso_lift_joint, arm_1_joint,
      arm_2_joint, arm_3_joint, arm_4_joint, arm_5_joint,
      arm_6_joint, arm_7_joint]
      points:
      - positions: [0.15, 0.20, -1.34, -0.20, 1.94, -1.57, 1.37, 0.0]
        time_from_start: 0.0
      - positions: [0.15, 0.20, -1.34, -0.20, 1.94, -1.57, 1.37, 0.0]
        time_from_start: 2.0
      - positions: [0.15, 0.21, -1.02, -0.20, 1.94, -1.57, 1.39, 0.0]
        time_from_start: 3.5
      - positions: [0.25, 0.07, 0.08, -0.20, 2.15, -1.57, 1.39, 0.0]
        time_from_start: 6.5
      - positions: [0.25, 0.07, 0.08, -0.20, 2.15, -1.57, 0.60, 0.0]
        time_from_start: 10.0
      - positions: [0.25, 0.45, 0.08, -1.5, 2.15, -1.57, 0.60, 0.0]
        time_from_start: 12.0
      - positions: [0.25, 0.45, 0.08, -1.5, 2.15, -1.57, 0.60, 1.5]
        time_from_start: 17

    pick_final_pose:
      joints: [torso_lift_joint, arm_1_joint,
      arm_2_joint, arm_3_joint, arm_4_joint, arm_5_joint,
      arm_6_joint, arm_7_joint]
      points:  
      - positions: [0.15, 0.45, 0.08, -1.5, 2.15, -1.57, -0.9, -1.5]
        time_from_start: 5.0
      - positions: [0.15, 0.45, 0.08, -1.5, 2.15, -1.57, 0.7, -1.5]
        time_from_start: 10.0
